chatt-app/ChatServer.java

```diff
@@ -1,93 +0,0 @@
-import java.io.*;
-import java.net.*;
-import java.util.StringTokenizer;
-import java.util.Vector;
-
-

```

Comment:
Description: Server that accepts connections from the chat client and properly handles them